perma_web/conftest.py

```diff
@@ -271,6 +271,36 @@ class Meta:
     password = factory.PostGenerationMethodCall('set_password', 'pass')
 
 
+@register_factory
+class RegistrarUserFactory(LinkUserFactory):
+    registrar = factory.SubFactory(RegistrarFactory)
+
+
+# SponsorshipFactory has to come after RegistrarUserFactory and LinkUserFactory,
+# and before SponsoredUserFactory
```

Comment:
Another way to deal with import problems like this, instead of worrying about the order of definition, is to use a string with the full path to the related fixture, instead of using the class itself, like [here](https://github.com/harvard-lil/perma/blob/develop/perma_web/conftest.py#L317).

E.g. something like `user = factory.SubFactory('conftest.LinkUserFactory)`, if this is above the definition of `LinkUserFactory` in the file.

This is perfect here; no need to change or move!

But since you commented, I thought I'd mention :-)